src/ansys/dpf/post/dataframe.py

```diff
@@ -833,3 +835,117 @@ def animate(
         return fc.animate(
             save_as=save_as, deform_by=deform_by, scale_factor=scale_factor, **kwargs
         )
+
+    def min(self, axis: Union[int, str, None] = 0) -> Union[DataFrame, float]:
+        """Return the minimum value over the requested axis.
+
+        Parameters
+        ----------
+        axis:
+            Axis to perform minimum across.
+            Defaults to the MeshIndex (0).
+            Can also be the SetIndex (1).
```

Comment:
Maybe here we could add additional info just to remind what are the `MeshIndex` and `SetIndex` for new users.
I don't remember if it is done somewhere else.